spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -854,9 +854,10 @@ def setup_editor(self,
         # Scrolling past the end
         self.set_scrollpastend_enabled(scroll_past_end)
 
-        # Line number area
+        # Line number area and indent guides
         if cloned_from:
             self.setFont(font) # this is required for line numbers area
+            self.patch = cloned_from.patch  # this is required for indentguides

```

Comment:
Since our tests fail due to this change, and we're not able to test this, let's enclose this line in an `if running_under_pytest():` so we can merge this PR.

Reply:
Also add a comment above this line with the following text:

```python
# NOTE: Please **do not** remove the line below because it'll break
folding and indent guides in splitted editors.
```